docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -241,16 +255,93 @@ To update an existing network with a new `blockperiodseconds`:
             }
             ```
 
-1. Restart all nodes in the network using the updated genesis file.
-1. To verify the changes after the transition block, call
+3. Restart all nodes in the network using the updated genesis file.
```

Comment:
@rolandtyler these get automatically numbered if you leave all the ordered list items as `1.`

Reply:
The version of IntelliJ I have has some auto feature that automatically renumbers them when I enter them as `1`.  There's still no release yet to fix it.  [Issue link](https://youtrack.jetbrains.com/issue/IDEA-273049).